tests/Test/Language/Souffle/InterpretedSpec.hs

```diff
@@ -55,7 +55,7 @@ spec = describe "Souffle API" $ parallel $ do
 
     it "returns just the program if it can load a souffle program" $ do
       prog <- Souffle.runSouffle $ do
-        Just handle <- Souffle.init Path
+        handle <- fromJust <$> Souffle.init Path
```

Comment:
`MonadFail` vs `error` although this is the same for the IO based implementations, am I right?

Reply:
Well, I think so.. but in a normal situation I would expect people to handle the `Nothing` case explicitly instead of doing `fromJust` (this is what is shown in the README as well).
In the tests I take a shortcut however :smile:.